userguide/tasks.rst

```diff
@@ -168,11 +168,18 @@ shows the configuration options for Cloud Syncs.
    | Enabled                | checkbox            | Unset to temporarily disable this Cloud Sync.                                                          |
    |                        |                     |                                                                                                        |
    +------------------------+---------------------+--------------------------------------------------------------------------------------------------------+
-   
-   
+
+
 .. note:: The time selected is when the Cloud Sync task is allowed to
    begin. The cloud sync runs until finished, even after the time selected.
 
+.. note:: If
+   `rclone sync <https://rclone.org/commands/rclone_sync/>`__
+   encounters any errors, files are not deleted in the destination.
```

Comment:
> If `rclone sync <https://rclone.org/commands/rclone_sync/>`__ encounters any errors, files are not deleted in the destination.

This is a little unclear to me. Are the "files" that are not deleted the files that were sent previously as part of the `rclone sync`, or other files on the destination system such as those from a previously completed rclone command?

Reply:
According to `rclone sync` docs: "Make source and dest identical, modifying destination only." What I have makes sense to me. Feel free to make a suggestion. I can't seem to think of a way to make it clearer.

Reply:
How about:
`
Files that have completed the sync process are not deleted from the destination if the rclone sync <link> is interrupted or encounters an error.
`
That feels a little more definitive.